airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/commands.py

```diff
@@ -3,22 +3,30 @@
 #
 
 import sys
-from typing import List
+from typing import Dict, List
 
 import asyncclick as click
 from pipelines import main_logger
 from pipelines.airbyte_ci.connectors.consts import CONNECTOR_TEST_STEP_ID
 from pipelines.airbyte_ci.connectors.context import ConnectorContext
 from pipelines.airbyte_ci.connectors.pipeline import run_connectors_pipelines
+from pipelines.airbyte_ci.connectors.test import argument_parsing
```

Comment:
❗Last thing in the journey to being generic. Lets move this to a generic location

```suggestion
from pipelines.airbyte_ci.helpers import argument_parsing
```

or better yet create a new area that denotes "These are things we use to execute the step tree"


```suggestion
from pipelines.helpers.execution.argument_parsing  import build_extra_params_mapping
from pipelines.helpers.execution.run_steps import RunStepOptions
```